docusaurus.config.js

```diff
@@ -0,0 +1,334 @@
+/* eslint-disable unicorn/no-array-push-push,global-require,@typescript-eslint/no-unused-vars */
+// noinspection JSUnusedLocalSymbols,JSUnusedGlobalSymbols
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+
+const darkCodeTheme = {
```

Comment:
I tried to put this in its own file, but I was getting errors like this:

```
[ERROR] /Users/gableroux/repos/gameci/game.ci/src/theme/dracula.js:110
export default theme;
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1033:15)
    at Module._compile (node:internal/modules/cjs/loader:1069:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gableroux/repos/gameci/game.ci/docusaurus.config.js:8:23)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at module.exports (/Users/gableroux/repos/gameci/game.ci/node_modules/import-fresh/index.js:32:59)
    at loadSiteConfig (/Users/gableroux/repos/gameci/game.ci/node_modules/@docusaurus/core/lib/server/config.js:21:55)
error Command failed with exit code 1.
```

Couldn't figure it out, but putting the theme in here directly as an object works just fine. Looks a bit noisy, but this is also something we can improve later.

Reply:
note that in Docusaurus v2 the `theme` folder is a special folder intended for [Swizzling](https://docusaurus.io/docs/swizzling). Did you swizzle the theme or just put this file there?

Reply:
we fixed this in a later change by using `modules.export` instead of `export default` 👍